### ISSUE TITLE
search history remove / disable

### ISSUE THREAD

**Issue by Klar** [mauriceoegerli/viewtube-vue#2451]
If you search something in the searchbox. And search it again next time it will show your search history and also the youtube suggestion overlapping.

I think it would make sense to only show the youtube suggestion and not the history of everything you have searched / written in the searchbox.

**Comment by frozenduck** [comment]
Could you be more specific as to how this problem manifests itself ? Perhaps a screenshot would be the best solution.

**Comment by moisout** [comment]
I think this is the built-in input field history of the browser

**Comment by Klar** [comment]
yes it seems that it is the input field from the browser.

If I click into the input field it first shows me the previous search (from the input field input). If I start typing it starts  youtube autocomplete.

I would say we could disable the input field autocomplete.

![2024-04-15_13-35](https://github.com/ViewTube/viewtube/assets/604278/8ff10214-c926-4a61-9b29-5e345b13424b)
![2024-04-15_13-35_1](https://github.com/ViewTube/viewtube/assets/604278/092c1deb-f71f-4dc7-8ad3-472bd48f1ec9)

**Comment by nargacu83** [comment]
> I would say we could disable the input field autocomplete.

I don't think that is necessary as people probably is used to that feature or might want to use that feature on ViewTube. I disabled it on my browser and don't have this problem. The easiest solution is to disable it in your browser or use an extension to set your viewtube instance as an exception.
